entities/TV.gd

```diff
@@ -0,0 +1,35 @@
+"""
```

Comment:
There're sources folder and entities, levels. Could you share me idea of changing structure?

Reply:
The idea is to give visibility on usefull things for non-devs: entities contains things that can be put into the level by the level designer and levels are the level that will actually be played.

I'm not really sure about this structure though because then, player should also be an entity :/ What do you think would be best?

Reply:
I like the idea that separate work of level design and coder.
I think we should keep it.
About player, and other cases, my first idea is create sub class that inherit player and contain no code. Only default exported variable(like speed...), graphic part can be changed(if applicable), no or limited core override function(which not related to graphic) is implemented. It's quite redundant. That's draft idea only. But I believe that we can find solution for it.
For now, let't see how much designer(level, graphic) need invoke on it(player) and compare with how much coder need invoke on it. I think coder need invoke more. So, still keep it in source